packages/core/src/hooks/use-account.ts

```diff
@@ -37,6 +35,19 @@ export type UseAccountResult = {
   status: AccountStatus;
 };
 
+/** 
+ * Validate and format the address.
+ *
+ * @param address - The address string to validate.
+ * @returns The validated and formatted address as Address type.
+ */
+function getAddress(address: string): Address {
+  return address.startsWith("0x")
```

Comment:
Can you use the [`validateAndParseAddress`](https://github.com/starknet-io/starknet.js/blob/develop/src/utils/address.ts#L41) function from starknet.js? Also move this function to a file `utils.ts` in `packages/core/src` and then export it from `packages/core/src/index.ts`